cilium/cilium.go

```diff
@@ -40,6 +44,83 @@ func New(config meshkitCfg.Handler, log logger.Handler, kc meshkitCfg.Handler) a
 	}
 }
 
+// ApplyOperation function contains the operation handlers
+func (h *Handler) ApplyOperation(ctx context.Context, request adapter.OperationRequest) error {
```

Comment:
Should I change structure name to be more specific `Cilium`??

Reply:
@MUzairS15, ideally, not. Anything that can be generic, should be generic so as to promote consistency across adapters. // @Revolyssup